api/controller/auth.js

```diff
@@ -87,4 +49,50 @@ return response.status(200).json({message:'User Successfully Logged In', info:to
 
 
 
-module.exports = {signup, login}
\ No newline at end of file
+
+
+
+
```

Comment:
Check your linting tools. I am having a challenge reading this file because it's not linted.